src/lib/image-upload-provider.ts

```diff
@@ -57,6 +63,9 @@ const handleClick = (cb: FileCallback): void => {
 const handleDrop = (event: DragEvent, cb: FileCallback): void =>
   cb(event.dataTransfer?.files?.[0])
 
+const handlePaste = (event: ClipboardEvent, cb: FileCallback): void =>
+  cb(event.clipboardData?.files?.[0])
```

Comment:
Same, no check on the file?

Reply:
yes the back-end will refuse a non-image (or directory) and a toaster will appear